test/e2e/set/scyllacluster/datainserter.go

```diff
@@ -0,0 +1,148 @@
+// Copyright (c) 2021 ScyllaDB
+
+package scyllacluster
+
+import (
+	"context"
+	"fmt"
+	"sort"
+	"strings"
+
+	"github.com/scylladb/scylla-operator/test/e2e/framework"
+
+	"github.com/gocql/gocql"
+	"github.com/scylladb/gocqlx/v2"
+	"github.com/scylladb/gocqlx/v2/table"
+	scyllav1 "github.com/scylladb/scylla-operator/pkg/api/scylla/v1"
+	"k8s.io/apimachinery/pkg/util/rand"
+	"k8s.io/apimachinery/pkg/util/uuid"
+	corev1client "k8s.io/client-go/kubernetes/typed/core/v1"
+)
+
+const nRows = 10
+
+type DataInserter struct {
+	client            corev1client.CoreV1Interface
+	session           *gocqlx.Session
+	keyspace          string
+	table             *table.Table
+	data              []*TestData
+	replicationFactor int
+}
+
+type TestData struct {
+	Id   int    `db:"id"`
+	Data string `db:"data"`
+}
+
+func NewDataInserter(ctx context.Context, client corev1client.CoreV1Interface, sc *scyllav1.ScyllaCluster, replicationFactor int) (*DataInserter, error) {
```

Comment:
nit: should we use int32 for RF so it avoids all the casts externally?

Reply:
changed, although to int64

Reply:
why don't we use the API type (int32)?

Reply:
int64 was initially used for counting members in a helper func, but I think you're right, there is no valid reason to keep it that way and cast every time it's used